modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -542,6 +542,9 @@ void GDScriptAnalyzer::resolve_class_interface(GDScriptParser::ClassNode *p_clas
 					if (datatype.type_source != GDScriptParser::DataType::UNDETECTED) {
 						datatype.type_source = GDScriptParser::DataType::INFERRED;
 					}
+					if (member.variable->initializer->type == GDScriptParser::Node::GET_NODE && !member.variable->onready) {
+						push_error(vformat(R"*(Use "@onready var %s = get_node(...)" instead.)*", member.variable->identifier->name), member.variable->initializer);
```

Comment:
I can't comment on whether the fix is correct, but the `*` used in the error message seem quite unusual compared to other errors?

Reply:
Yeah the *'s there because otherwise the `)"` after `get_node(...)"` closes the error string causing an error. The *s on both side prevent that from happening.

Reply:
Look at line 1208 here : https://www.github.com/godotengine/godot/tree/master/modules%2Fgdscript%2Fgdscript_parser.cpp it employs a similar format

Reply:
It's the magic of C++11 raw strings letting you use any delimiter :slightly_smiling_face: